### PR TITLE
Fix for GCC 9 and clang 10 

### PR DESCRIPTION
**Pull request by MichalLeonBorsuk**
Modern compilers won't compile "goto done_close" claiming code jumps over a variable definition. 
Fix: copied code inline.

### CODE REVIEW

**Comment by bjorg** [comment]
Wow. Blast from the past! How did you even find this repo? It's ancient and has no readme file. Did you use it for anything? Just curious. And thanks for the pull request!

File: main.cpp [hunk 1]
```diff
@@ -242,7 +242,9 @@ int main (int argc, char * const argv[]) {
 	FLDBHeader header;
 	if(!header.Read(f)) {
 		printf("ERROR: '%s' is not an FLDB file\n", db_filename);
-		goto done_close;
+    	fclose(f);
+	    f = nullptr;
```
Comment (bjorg) [inline main.cpp:246]: If it's not too much to ask, could you fix the indentation for this line and the next? Thx!

Comment (MichalLeonBorsuk) [reply main.cpp:246]: Sure. I guess one of my editors still has tab in place of spaces...

**Comment by MichalLeonBorsuk** [comment]
Someone in a forum post 10 years ago linked to your prject. This format is still in use as on 2021.

**Comment by bjorg** [comment]
@MichalLeonBorsuk  Why did you close the pull-request?

**Pull request closed without merging**